client/guidestours/steps.js

```diff
@@ -38,10 +38,10 @@ class GuidesFirstStep extends Component {
 
 		const { text, onNext, onQuit } = this.props;
 		return (
-			<Card className="guidestours__step" style={ stepCoords } >
+			<Card className="guidestours__step guidestours__step-first" style={ stepCoords } >
```

Comment:
Is this always going to be a first step? Should we rename the class, or take in an `isFirst` prop?